### PR TITLE
SelectableItemBackground Ripple!

### PR DESCRIPTION
**Pull request by MarcRubio**
Adds ripple effect with tiny addition to code.

### CODE REVIEW

**Comment by yombunker** [comment]
@MarcRubio i'll try this branch tonight to make sure no behavior breaks with this change 👍

File: bottom-bar/src/main/java/com/roughike/bottombar/BottomBarTab.java [hunk 1]
```diff
@@ -92,6 +93,11 @@ void prepareLayout() {
         setGravity(Gravity.CENTER_HORIZONTAL);
         setLayoutParams(new LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT));
 
+        // set a selectable item background
+        final TypedValue typedValue = new TypedValue();
```
Comment (roughike) [inline bottom-bar/src/main/java/com/roughike/bottombar/BottomBarTab.java]: @yombunker @MarcRubio 

See the MiscUtils.java, which has a [pretty similar method for resolving attributes](https://github.com/roughike/BottomBar/blob/master/bottom-bar/src/main/java/com/roughike/bottombar/MiscUtils.java#L38-L42).

The method is incorrectly named to ```getColor```, but should probably be ```getResourceAttribute```, and the **@ColorRes** annotation should be replaced with **@AttrRes** to get rid of the lint warning in the BottomBar class.

See if you could make that method reusable for this and resolving the primary color, so this wouldn't have to be done inline.

**Comment by yombunker** [comment]
@MarcRubio @roughike does the square look good to you guys? i tried with borderless and i think it looks a little better, but, want to know your opinion about this. or if we can't agree, then we need to allow the user to set whatever he thinks is the best for their app

**Comment by roughike** [comment]
That's a good catch. It was originally the borderless selectableitembackground in the v1, and I see no reason to not make it that way on this version as well.

**Comment by roughike** [comment]
Should we include this in the next version, whatever it will be?

With the selectableItemBackgroundBorderless background.

**Comment by yombunker** [comment]
@roughike @MarcRubio i think this should be added in the next version, but, we need @MarcRubio to fix the small comment before merging, or we could just merge it and fix it ourselves :)

**Comment by MarcRubio** [comment]
Hey sorry for slightly late reply, been kinda busy. Hope the next commit is what you wanted ;)

**Pull request merged**